content/en/blog/2024/otel-errors/index.md

```diff
@@ -0,0 +1,286 @@
+supports [`RecordError`](https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/trace/span.go#L445-L467), which essentially does the same thing idiomatically. You have to 
+make an additional call to set its status to `Error` if that’s what it should be, 
+as it won’t automatically be set to that. Similarly, `RecordException` can be used 
+to record span events without setting the span’s status to ERROR, which means you 
+can use it to record any additional data about a span. 
+
+By decoupling the span status from being automatically set to `Error` when a span 
+exception occurs, you can support the use case where you can have an exception 
+event with a status of `Ok` or `Unset`. This gives instrumentation authors the 
+most flexibility. 
+
+### Errors Depicted in Logs
+In OTel, a log is a structured, timestamped message emitted by a service or other 
+component. The recent addition of logs to OTel gives us yet another way of 
+reporting errors. Logs have traditionally had different severity levels for 
+representing the type of message being emitted such as `DEBUG`, `INFO`, `WARNING`, 
+`ERROR`, and `CRITICAL`.
+
+OTel allows for the correlation of logs to traces, in which a log message can be 
+associated to a span within a trace, via trace context correlation. Hence, looking 
+for a log message with a log level of ERROR or CRITICAL can yield further 
+information of what led to that error, by pulling up the correlated trace.
+
+To record an error on a log, either `exception.type` or `exception.message` is 
+required, while `exception.stacktrace` is recommended. You can view more 
+information about the semantic conventions for log exceptions [here](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-logs.md).
+
+## Logs or Spans to Capture Errors?
+After all this, you might be wondering which signal to use to capture errors: 
+spans or logs? The answer is: "It depends!" Perhaps your team primarily uses 
+traces; perhaps they primarily use logs.
+
+Spans can be great for capturing errors, because if the operation errors out, 
+marking a span as an error makes it stand out and therefore easier to spot. On 
+the other hand, if you’re not filtering or tail sampling your traces and your 
+system is producing thousands of spans per minute, you could miss errors that 
+aren’t occurring frequently, but that still need to be handled. 
+
+What about using span events versus logs? Again, this depends. It may be 
+convenient to use span events, because when a span status is set to `Error`, a 
+span event with the exception message (and other metadata you may wish to capture) 
+is automagically created. 
+
+Another consideration is your observability backend. Does your backend render 
+both logs and traces? How easily queryable or discoverable are your logs, spans, 
+and span events? Is logs and trace correlation supported?
+
+## Visualizing Errors in Different Backends:
```

Comment:
```suggestion
## Visualizing errors in different backends

```

Reply:
A few notes on this section in general:

* provide higher resolutions for the screenshots attached to the blog post, I can barely see anything on them.
* as always I have mixed feelings about putting screenshots from commercial vendors into a blog post. I don't think it's much of a concern for the KubeCon talk, but we try to keep the vendor-neutrality high. Do you think it subtracts a lot of value from this blog post if we remove them and keep jaeger only?

Reply:
* Yes, I became aware of how low-res they were once I saw the preview; we will replace them with hi-res images.
* We also debated on screen shots from vendors. We do think it's something that's of value to end users, as it shows how the same OTel error is expressed in different backends. We took care to remove any mention of the vendors and keep them anonymous (although we are aware some people may be able to guess). We tried to keep it as factual as possible and not pitch anything. Of course, we are happy to remove them altogether if you feel strongly against it, but wanted to open a convo about it first @svrnm

Reply:
One idea was, if it'd be helpful and is supported, to shorten this section by using a carousel for the images. That way readers have to click through the images to view them, and we can place Jaeger or use a cover image as the first image so the vendor screen shots are not explicitly shown. Thoughts?

Reply:
I understand where you are coming from, but including vendor material (in this ), is always something we need to be careful about, even if it is anonymized.

We have 2 options here: 
* you remove that section, or reduce it to a minimum (show the jaeger part, and add a pure textual section that different backends may behave differently) and we are fine.
* we discuss what needs to be done to be in line with [our guidelines](https://github.com/open-telemetry/community/blob/main/social-media-guide.md). As the argument you make is that this falls under "It's OK to refer to a specific product or service in the context of a technical discussion, but avoid promoting a specific product or service over others.". In this case I'd like to have consensus among @open-telemetry/docs-maintainers that we are good here.

I am fine either way, just let me know what you prefer to do

Reply:
re-opening this, to double check what we want to do here. 

@open-telemetry/docs-maintainers PTAL as well

Reply:
i feel like this is is one of those "if you didn't bring enough for the whole class..." situations. 

> you remove that section, or reduce it to a minimum (show the jaeger part, and add a pure textual section that different backends may behave differently) and we are fine.

I'd be OK with this; discuss how different tools offer different options, but don't show them.

Reply:
Thank you both! Adriana and I decided to go with Severin's first option, so I'll make those changes shortly.

Reply:
thanks!

Reply:
I reworked this entire section. Can you lmk your thoughts?

Reply:
I uploaded the images via the GH UI and I don't know how to remove them

Reply:
if you click on the `...` besides the images on the Files view of this PR you _might_ be able to delete them. If not, you need to pull the PR locally and remove them (I use the gh cli tool for that, which allows you to do `gh pr checkout 4105`), let me know if I should do that for you

Reply:
> I reworked this entire section. Can you lmk your thoughts?

looks good now, had only 1 minor remark

Reply:
@svrnm Can you please remove the images for me if it's not an inconvenience 🙏 Thank you!

Reply:
Nvm I managed to do it!